src/components/organizations/index.jsx

```diff
@@ -1,26 +1,81 @@
-import {useEffect, useState } from "react";
-import { getOrganizations } from "../../api/endpoints";
+import { useEffect, useState } from "react";
 import OrganizationsList from "../common/OrganizationsList";
-import {Box, Typography, CircularProgress} from "@mui/material";
+import { Box, Typography, CircularProgress, Stack, Button, Link } from "@mui/material";
+import BasicDialog from "../common/BasicDialog";
+import GroupAddOutlinedIcon from "@mui/icons-material/GroupAddOutlined";
+import { createNewOrganization, getOrganizations } from "../../api/endpoints/apiService";
 
 import { vars } from "../../theme/variables";
-const { gray700 } = vars;
+const { gray600, gray700, brand700, brand800 } = vars;
+
+const linkStyles = {
+  color: brand700,
+  fontWeight: 600, 
+  textDecoration: "none", 
+  "&:hover": { 
+    color: brand800 
+  }
+}
 
 const Organizations = () => {
   const [organizations, setOrganizations] = useState([]);
-  const [loading, setLoading] = useState(true);
+  const [loading, setLoading] = useState(false);
+  const [open, setOpen] = useState(false);
+  const [message, setMessage] = useState();
+
+  // TODO: change this to be dynamic when we get the response from api call
+  const groupname = "aigul"
 
   const fetchOrganizations = async() => {
-    const organizations = await getOrganizations()
-    setOrganizations(organizations);
-    setLoading(false)
+    setLoading(true);
+  
+    try {
+      const response = await getOrganizations(groupname)
+      if(response.length > 0){
+        setOrganizations(response)
+      }
+    } catch (err) {
+      console.error('An unknown error occurred: ', err);
+    } finally {
+      setLoading(false);
+    }
   }
 
   useEffect( () => {
     setLoading(true)
     fetchOrganizations();
   }, []);
 
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const createOrganization = async (event) => {
```

Comment:
@Aiga115 same for this function here, it is the same of the other file, same comment. Thanks